### PR TITLE
[gcc7] add symbolic links, recompile packages

### PR DESCRIPTION
**Pull request by lovedheart**

### CODE REVIEW

File: packages/gcc7.rb [hunk 1]
```diff
@@ -125,8 +130,29 @@ def self.install
...
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/gcov-tool-7.3 #{CREW_DEST_PREFIX}/bin/gcov-tool"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/gfortran-7.3 #{CREW_DEST_PREFIX}/bin/gfortran"
+      
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-c++-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-c++"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-g++-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-g++"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ar-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ar"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-nm-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-nm"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ranlib-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gcc-ranlib"
+      system "ln -sv #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gfortran-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gfortran"
```
Comment (uberhacker) [inline packages/gcc7.rb]: The source of all the symlinks should be `#{CREW_PREFIX}` instead of `#{CREW_DEST_PREFIX}`.  Target is fine.

Comment (lovedheart) [reply packages/gcc7.rb]: Change all #{CREW_DEST_PREFIX} to #{CREW_PREFIX} ?

Comment (uberhacker) [reply packages/gcc7.rb]: Only the source.  Target is fine.  Example:
```
system "ln -sv #{CREW_PREFIX}/bin/$(gcc -dumpmachine)-gfortran-7.3 #{CREW_DEST_PREFIX}/bin/$(gcc -dumpmachine)-gfortran"
```

Comment (lovedheart) [reply packages/gcc7.rb]: @uberhacker fixed

**Comment by uberhacker** [comment]
Did you recompile and create new pre-built binaries after the symlink changes?

**Comment by lovedheart** [comment]
@uberhacker yes

**Pull request merged**